addons/l10n_ch/models/account_invoice.py

```diff
@@ -296,8 +300,6 @@ def isr_print(self):
     def print_ch_qr_bill(self):
         """ Triggered by the 'Print QR-bill' button.
         """
-        self.ensure_one()
```

Comment:
removing the `ensure_one` is weird regarding the access to specific fields on `self` above